topics/entity-relationships.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: site
+title: Entity Relationship Diagram
+body_class: entity-relationships
+mermaid: true
+---
+
+There are a lot of things within the Talis Aspire eco system! The diagram at the bottom of this page represents the key relationships between the main tables in the Advanced MIS database.
+
+## Relationships
+
+In order to interpret the diagram it is useful to know the type of relationship between two tables. These are expressed as follows.
+
+* ... Dotted lines mean there is a non-identifying relationship, i.e. the two entities can co-exist independently of the other
+* --- Solid lines mean there is an identifying relationship where the entity cannot exist without other referenced entities also existing.
+
+| (left) | (right) | Meaning |
+| - | - | - |
+| `|o` | `o|` | Zero or one |
+| `||` | `||` | Exactly one |
+| `}o` | `o{` | Zero or more (no upper limit)
+| `}|` | `|{` | One or more (no upper limit)
+
+Labels on relationships indicate where there is a reliance on a particular key being present.
+
+## Attributes
+
+We have not listed all attributes of a table, only those that can be used as either primary of foreign keys.
+
+<div class="mermaid">
```

Comment:
This DIV is rendered by mermaid-js to become the ERD diagram.  
Things to check here are attribute/table names being correct.